docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -16,37 +19,100 @@ For scripts to successfully use BODL to access storefront data, the following mu
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | Items added to the cart. | See [line_item](#line_item-object-schema) object for event fields |
+| `product_value` | number | Item price before discounts and coupons. Product default price or, if applicable, the sale price configured in the admin panel. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.physicalItems.listPrice` or <br /> `lineItems.digitalItems.listPrice`|
+
+### View cart
+
+When the shopper clicks a button to view a cart, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `cart_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `cartAmount` |
+| `channel_id` | string | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | All items in the cart. | See [line_item](#line_item-object-schema) object for event fields |
+
+### Remove product from cart
+
+When the shopper clicks a button to remove a product from the cart, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `cart_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `cartAmount` |
+| `channel_id` | string | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `line_items[]` | array of `line_item` objects | Items removed from the cart. | See [line_item](#line_item-object-schema) object for event fields |
```

Comment:
As per [the doc](https://docs.google.com/document/d/13qlYa9gtuVcjs2iBZ-jWP74W1GPRQw9J496t7QYbbJg/edit#heading=h.d4f6ozfxng47), there should be event_id also, but it's missing in reality
@bc-sanjayjoshi

Reply:
@bc-rmalyavc event_id is not needed for GA4 but from the BODL perspective, it is needed as we expand to other tech partners.

Reply:
@bc-sanjayjoshi Yes, this doc is about BODL. 

The `event_id` is currently missing from BODL when I test it in my browser. If we're planning on fixing that, it would be great if you could provide an engineering ticket that I can watch, so I can update my doc once BODL is fixed